android-studio

```diff
@@ -34,10 +34,14 @@
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://dl.google.com/dl/android/studio/ide-zips/$version/android-studio-ide-$matchBuild-windows.zip"            },
+                "url": "https://dl.google.com/dl/android/studio/ide-zips/$version/android-studio-ide-$matchBuild-windows.zip"
+            },
             "32bit": {
                 "url": "https://dl.google.com/dl/android/studio/ide-zips/$version/android-studio-ide-$matchBuild-windows32.zip"
             }
+        },
+        "hash": {
+            "url": "https://developer.android.com/studio/"
```

Comment:
There are some sha256 in the source of the page. Search for "COPY THIS FOR ARCHIVE PAGE" and you will find them. If they get removed, it will fallback to simply downloading the files :smile: